dotnetv3/Lambda/scenarios/Lambda_Basics/Lambda_BasicsTests/LambdaMethodsTests.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier:  Apache-2.0
+
+using Newtonsoft.Json.Linq;
+using Xunit.Extensions.Ordering;
+
+namespace Lambda_Basics.Tests
+{
+    public class LambdaMethodsTests
+    {
+        private readonly IConfiguration _configuration;
+        private readonly AmazonLambdaClient _client;
+        private readonly LambdaMethods _LambdaMethods;
+
+        public LambdaMethodsTests()
+        {
+            _client = new AmazonLambdaClient();
+            _configuration = new ConfigurationBuilder()
+                .SetBasePath(Directory.GetCurrentDirectory())
+                .AddJsonFile("testsettings.json") // Load test settings from JSON file.
+                .AddJsonFile("testsettings.local.json",
+                true) // Optionally load local settings.
+            .Build();
+
+            _LambdaMethods = new LambdaMethods();
+        }
+
+        [Fact()]
+        [Order(1)]
+        public async Task CreateLambdaFunctionTest()
+        {
+            var functionArn = await _LambdaMethods.CreateLambdaFunction(
+                _client,
+                _configuration["FunctionName"],
+                _configuration["BucketName"],
+                _configuration["Key"],
+                _configuration["Role"],
+                _configuration["Handler"]);
+            Assert.NotNull(functionArn);
+        }
+
+        [Fact()]
+        [Order(6)]
```

Comment:
If possible, I would order these in the order you want them to run, or as close as possible. But it's fine if they can't be reordered. It would simply be cleaner if the Order(#) lines were also in order.